### PR TITLE
D8core-2765 Styles and icons for mailto buttons

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- fixes mailto links on buttons
https://stanfordits.atlassian.net/browse/D8CORE-2765

# Needed By (Date)
soon

# Urgency
Not urgent

# Steps to Test

1. Check out this branch and clear cache
2. create buttons with with mailto links. There are 3 types of buttons.
3. Verify appropriate color of mailto link icon

# Affected Projects or Products
-  D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2765
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@ishahism 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss [hunk 1]
```diff
@@ -80,10 +80,10 @@ a {
     }
   }
 
-  svg {
 
-    // Style the SVG for mailto.
-    &.mailto {
+  // Style the SVG for mailto.
+  &.mailto {
```
Comment (pookmish) [inline modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: This seems fine to me, but the styles aren't in the wysiwyg. Do we want to work on that or we can make that another ticket.

**Review by pookmish — APPROVED** [review]

File: modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss [hunk 2]
```diff
@@ -95,6 +95,38 @@ a {
...
+
+    &.su-button--big {
+      svg {
+        fill: $su-color-white;
+      }
+
+      path {
+        stroke: $su-color-white;
+      }
+    }
```
Comment (pookmish) [inline modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:119]: These can be combined

File: modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss (hunk 1 above)
Comment (cjwest) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: Thanks for your comment, @pookmish. 
 If I create a button through the WYSIWYG and use an email link, I see the icon show up with the color change. Please help me understand when you say the styles aren't in the WYSIWYG?

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: image

Comment (cjwest) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: @pookmish,

Ahhh, that makes sense. Where do we go to make this magic happen?

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: https://github.com/SU-SWS/stanford_profile_helper/blob/8.x-1.x/modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss
The issue is that the classes and stuff generated by the external links module won't be in the wysiwyg. You might have to fake it.

Comment (cjwest) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: How about I start a new PR for the WYSIWYG styles?

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: that sounds good. Please create a ticket as well.

**Review by pookmish — APPROVED** [review]

File: modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss (hunk 1 above)
Comment (cjwest) [reply modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss:85]: https://stanfordits.atlassian.net/browse/D8CORE-2950

**Pull request merged**